### PR TITLE
Adds calendso to Booking and Scheduling

### PR DESCRIPTION
**Pull request by asm0dey**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [ x Additions are not already listed at any of
  - [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management),
  - [awesome-analytics](https://github.com/onurakpolat/awesome-analytics) (analytics),
  - [staticgen.com](https://www.staticgen.com/)
  - [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] The `Language` tag is the main server-side requirement for the software - don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1291,6 +1291,7 @@ _See also [Monitoring](#monitoring)_
 
 - [Alf.io](https://alf.io/) - The open source ticket reservation system. ([Demo](https://demo.alf.io/authentication), [Source Code](https://github.com/alfio-event/alf.io)) `GPL-3.0` `Java`
 - [Easy!Appointments](http://easyappointments.org/) - A highly customizable web application that allows your customers to book appointments with you via the web. ([Demo](http://easyappointments.org/demo/), [Source Code](https://github.com/alextselegidis/easyappointments)) `GPL-3.0` `PHP`
+- [calendso](https://calendso.com/), [Source Code](https://github.com/calendso/calendso) -  The open-source Calendly alternative.  ([Demo](https://app.calendso.com/bailey)) `MIT` `Nodejs`
```
Comment (nodiscc) [inline README.md:1294]: Please fix formatting

```
- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` 
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (asm0dey) [reply README.md]: Fixed, sorry for inconvinience

**Comment by asm0dey** [comment]
Excuse me, I don't understand why does it fail :(

**Comment by marcelbrueckner** [comment]
> Excuse me, I don't understand why does it fail :(

Your addition contains duplicate spaces. I've inserted `_` below at the positions where you placed them, two in total. If you remove them, the pipeline will probably run through.

```markdown
- [calendso](https://calendso.com/) - _The open-source Calendly alternative. _([Demo](https://app.calendso.com/bailey), [Source Code](https://github.com/calendso/calendso)) `MIT` `Nodejs`
```

**Comment by asm0dey** [comment]
Thank you for pointing this to me!

File: README.md [hunk 2]
```diff
@@ -1291,6 +1291,7 @@ _See also [Monitoring](#monitoring)_
 
 - [Alf.io](https://alf.io/) - The open source ticket reservation system. ([Demo](https://demo.alf.io/authentication), [Source Code](https://github.com/alfio-event/alf.io)) `GPL-3.0` `Java`
 - [Easy!Appointments](http://easyappointments.org/) - A highly customizable web application that allows your customers to book appointments with you via the web. ([Demo](http://easyappointments.org/demo/), [Source Code](https://github.com/alextselegidis/easyappointments)) `GPL-3.0` `PHP`
+- [calendso](https://calendso.com/) - The open-source Calendly alternative. ([Demo](https://app.calendso.com/bailey), [Source Code](https://github.com/calendso/calendso)) `MIT` `Nodejs`
```
Comment (prayashm) [inline README.md:1294]: Let's order them:
```suggestion
- [Calendso](https://calendso.com/) - The open-source Calendly alternative. ([Demo](https://app.calendso.com/bailey), [Source Code](https://github.com/calendso/calendso)) `MIT` `Nodejs`
- [Easy!Appointments](http://easyappointments.org/) - A highly customizable web application that allows your customers to book appointments with you via the web. ([Demo](http://easyappointments.org/demo/), [Source Code](https://github.com/alextselegidis/easyappointments)) `GPL-3.0` `PHP`
```

**Review by nodiscc — CHANGES_REQUESTED** [review]
> The open-source Calendly alternative

I have no idea about what calendly is/does and therefore no idea of what calendso is/does. Please clarify the description

**Comment by asm0dey** [comment]
@nodiscc done

**Review by nodiscc — APPROVED** [review]

**Pull request merged**